project/REVIEWING.md

```diff
@@ -220,7 +220,7 @@ review session. The goal of that session is to agree on one of the following out
  * Continue, either by pushing the PR further in the workflow, or by deciding to carry the patch
    (ideally, a maintainer should be immediately assigned to make sure that the PR keeps continued
    attention)
- * Escalate to Solomon by formulating a few specific questions on which his answers will allow
+ * Escalate to Solomon by formulating a few specific questions on which their answers will allow
```

Comment:
This is about a specific person, and using their preferred pronoun (at least, at the time of writing)

But we should remove these bullets as there's no longer a BDFL